schema/schema.go

```diff
@@ -189,7 +189,7 @@ func applyAllMigrationsToSchema(schema *serializers.Schema) error {
 	versions := migrationRepo.SortedVersions()
 
 	for _, version := range versions {
-		err = migrationRepo[version].ApplyToSchema(schema, migrationRepo)
+		err = migrationRepo[version].ApplyToSchema(schema, migrationRepo, false)
```

Comment:
This func is used from `generate` and `generate` should validate that the migrations apply cleanly, so we pass false for `idempotently`

Reply:
makes sense